gm4_liquid_tanks/data/gm4_liquid_tanks/function/item_process.mcfunction

```diff
@@ -2,7 +2,7 @@
 #run from liquid_tanks:process
 
 # get stack count for smart interface
-execute store result score $stack_size gm4_lt_value run data get storage gm4_liquid_tanks:temp/tank input_slot.count 1.0
+execute store result score $stack_size gm4_lt_value run data get block ~ ~ ~ Items[{Slot:0b}].count 1.0
```

Comment:
To avoid NBT, you can use `execute store ... if items block ~ ~ ~ container.0 *` to get the count